pkg/sippyserver/metrics/metrics.go

```diff
@@ -30,6 +30,11 @@ const (
 	blockerScoreToAlertOn = 70
 )
 
+var (
+	// CRTimeRoundingFactor will be synced with CRTimeRoundingFactor from Command Option
+	CRTimeRoundingFactor = 0 * time.Hour
```

Comment:
Can you clarify this comment? I think setting this to zero makes metrics hit component readiness every 5 minutes 💸 💸 💸

Reply:
This parameter is always synced with the command argument --component-readiness-time-rounding-factor. People can intentional disable rounding by setting --component-readiness-time-rounding-factor=0. When they do, this variable stays the default value of 0s and the old logic (24 hours one) will kick in.

Reply:
Ah I see, can you add a comment that 0 = disabled. I see the `>` check below now